src/mkdocs_autorefs/references.py

```diff
@@ -164,9 +165,11 @@ def inner(match: Match):  # noqa: WPS212,WPS430
                 return f"[{identifier}][]"
             return f"[{title}][{identifier}]"
 
+        classes = ["autorefs", "autorefs-external" if urlparse(url).scheme else "autorefs-internal"]
```

Comment:
Should we use `urlsplit` as well here and check for `netloc`?

Reply:
Yes actually...